web/src/App.tsx

```diff
@@ -20,6 +21,8 @@ function AppRoutes() {
     { path: "/dashboard/:workspaceId", element: <DashboardPage /> },
     { path: "/workspaces/:workspaceId/:projectId", element: <ProjectPage /> },
     { path: "/workspaces/:workspaceId/members", element: <MembersPage /> },
+    { path: "/workspaces/:workspaceId/:projectId/schema", element: <SchemaPage /> },
+    { path: "/workspaces/:workspaceId/:projectId/schema/:modelId", element: <SchemaPage /> },
```

Comment:
https://stackoverflow.com/a/53074599/13699142

Reply:
@KaWaite , It takes a string inside useRoute hook, what's your idea here exactly?

Reply:
Oh sorry I didn't realize there was a v6 update to the comment. My bad